src/components/explore/ProbeExplorer.svelte

```diff
@@ -42,7 +46,12 @@
   export let yScaleType;
   export let yDomain;
   export let densityMetricType;
-  export let yTickFormatter = format(',d');
+  export let yTickFormatter = formatCount;
```

Comment:
Maybe it would be worth creating a reusable function for assigning a formatter based on metric type?